Components/Alerts/Alert.css

```diff
@@ -0,0 +1,56 @@
+@import url(/Styles.css);
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700;900&display=swap');
+
+.Flex-Coloumn{
+    flex-direction: column;
+}
+
+.Alert{
+    width: 100%;
+    padding: 1rem;
+    border-radius: 0.35rem;
+}
+
+.Alert-Primary-Filled{
+    color: var(--primary-color);
+    background-color: #7FD2E6;
+}
+
+.Alert-Error-Filled{
+    color: var(--error-color);
+    background-color: #FCC5C5;
+}
+
+.Alert-Success-Filled{
+    color: var(--success-color);
+    background-color:#84F1C9 ;
+}
+
+.Alert-Warning-Filled{
+    color: var(--warning-color);
+    background-color:#FFE7C5 ;
```

Comment:
Don't hard code color value use as a variable

Reply:
I was thinking of it but these colors are only going to be used here and in no other component, hence hardcoded.

Reply:
So create :root CSS pseudo-class inside this file alert.css and define all color inside that and use it.